pandas/core/arrays/string_.py

```diff
@@ -265,6 +265,48 @@ def value_counts(self, dropna=False):
 
         return value_counts(self._ndarray, dropna=dropna).astype("Int64")
 
+    def min(self):
+        """
+        Return the minimum value.
+
+        Returns
+        -------
+        scalar
+            Minimum value.
+
+        See Also
+        --------
+        ExtensionArray.max : Return the maximum value of the object.
```

Comment:
why is this overriden? what is different?

Reply:
`StringArray` is the subclass of `PandasArray` which has its own `min` and `max` methods using nanops.

The class hierarchy is
`StringArray` <--- `PandasArray` <--- `ExtensionArray`.

The methods we add are in `ExtensionArray`. So these methods cannot be inherited.

Reply:
ok, can we use the same doc-string though?

Reply:
if *may* be worth it to define min, max in the base class as _min/_max (and just call them here) to avoid duplicating no?